nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service/src/main/java/org/apache/nifi/hbase/HBase_1_1_2_ClientService.java

```diff
@@ -88,16 +88,16 @@
 
     static final long TICKET_RENEWAL_PERIOD = 60000;
 
-    private volatile Connection connection;
+    protected volatile Connection connection;
```

Comment:
Can we leave the connection as private and add a `protected Connection getConnection()` for the lookup service to use?